src/test/java/com/zufar/onlinestore/user/api/SaveUserOperationPerformerTest.java

```diff
@@ -0,0 +1,57 @@
+package com.zufar.onlinestore.user.api;
+
+import com.zufar.onlinestore.openapi.dto.UserDto;
+import com.zufar.onlinestore.user.converter.UserDtoConverter;
+import com.zufar.onlinestore.user.entity.UserEntity;
+import com.zufar.onlinestore.user.repository.UserRepository;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(SpringExtension.class)
+class SaveUserOperationPerformerTest {
```

Comment:
I think it would be great if we will be adding some description for test classes
for example:
@DisplayName("Save user operation perfomer tests")
it is useful when we read logs while building